### PR TITLE
starting to turn analytics_single into a single-box sandbox playbook

### PR DESCRIPTION
**Pull request by shnayder**
Added missing hive metastore config. 
Removed unneeded EDXAPP config. 
Turned edx services back on.

@mulby @e0d

### CODE REVIEW

**Comment by mulby** [comment]
:+1:

File: playbooks/vagrant-fullstack.yml [hunk 1]
```diff
@@ -40,4 +40,4 @@
     - certs
     - role: analytics_api
       when: ANALYTICS_API_GIT_IDENTITY
-    - edx_ansible
+    - edx_ansible    # TODO: why is this line repeated? Remove if not needed, comment otherwise.
```
Comment (feanil) [inline playbooks/vagrant-fullstack.yml]: Shouldn't need this twice, let's remove it.

**Comment by feanil** [comment]
:+1:

File: playbooks/vagrant-fullstack.yml (hunk 1 above)
Comment (shnayder) [inline playbooks/vagrant-fullstack.yml]: removed

**Comment by jibsheet** [comment]
Unfortunately, this appears to be breaking Continuous Integration
http://jenkins.edx.org:8080/job/ansible-provision-continuous-integration/18283/console

Relevant bit below (looks like something isn't getting substituted?).

failed: [int.sandbox.edx.org] => (item={'db': u'{# HIVE_METASTORE_DATABASE.name | default(None) #}', 'user': u'{# HIVE_METASTORE_DATABASE.user | default(None) #}', 'pass': u'{# HIVE_METASTORE_DATABASE.password | default(None) #}'}) => {"failed": true, "item": {"db": "{# HIVE_METASTORE_DATABASE.name | default(None) #}", "pass": "{# HIVE_METASTORE_DATABASE.password | default(None) #}", "user": "{# HIVE_METASTORE_DATABASE.user | default(None) #}"}}
01:12:26 msg: (1470, "String '{# HIVE_METASTORE_DATABASE.user | default(None) #}' is too long for user name (should be no longer than 16)")

**Pull request merged**